micrometer-core/src/test/java/io/micrometer/core/aop/MeterTagSupportTests.java

```diff
@@ -15,74 +15,14 @@
  */
 package io.micrometer.core.aop;
 
-import io.micrometer.common.annotation.ValueExpressionResolver;
-import io.micrometer.common.annotation.ValueResolver;
-import io.micrometer.core.annotation.Timed;
-import org.junit.jupiter.api.Test;
-
-import java.lang.annotation.Annotation;
-import java.lang.reflect.Method;
-
-import static org.assertj.core.api.Assertions.assertThat;
-
 /**
- * Tests for {@link MeterTagSupport}.
+ * Tests for {@link MeterTagSupport} have been moved to the module
+ * micrometer-samples-spring-framework6 to test with the current version of Spring that
+ * requires JDK 17+..
  *
  * @author Marcin Grzejszczak
  * @author Johnny Lim
  */
 class MeterTagSupportTests {
```

Comment:
We could just delete this file, but I thought it might help to have a reference in the same project to the test class for the main code.